phpBB/install/startup.php

```diff
@@ -81,4 +81,5 @@ function phpbb_include_updated($path, $phpbb_root_path, $optional = false)
 
 $phpbb_installer_container = $phpbb_installer_container_builder
 	->with_config_path($config_path)
+	->with_custom_parameters(array('cache.driver.class' => 'phpbb\cache\driver\file'))
```

Comment:
This is just the installer so there are things that are really more important than manually specifying the cache type.